plugins/explore/README.md

```diff
@@ -104,11 +104,11 @@ Create a custom explore page in
 
 ```tsx
 import {
-  DomainExplorerContent,
+  CatalogKindExplorerContent,
```

Comment:
This should be CatalogKindExploreContent without the "r" (Explore, not Exporer). Same below

Reply:
Good catch, thank you!